cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/Cql2SqlUtilTest.java

```diff
@@ -76,10 +76,29 @@ public Object cql2likeParams() {
 
   @Test
   @Parameters(method = "cql2likeParams")
-  public void cql2like(String cql, String sql) {
+  public void quotedString(String cql, String sql) {
     assertThat(Cql2SqlUtil.cql2like(cql), is(sql));
   }
+  public Object quotedStringParams() {
+    return params(
+        "           ",
+        "'     ''   ",
+        "a     a    ",
+        "*     %    ",
+        "?     _    ",
+        "\\    \\\\ ",
+        "\\*   \\*  ",
+        "\\?   \\?  ",
+        "\\'   ''   ",
+        "\\\\  \\\\ "
```

Comment:
There is no backslash escape in an SQL string.

Correct:

    "\\\\  \\ ",

The CQL string "\\" represents a single backslash. In CQL it needs to be prepended by a masking backslash.
The SQL string '\' represents a single backslash.